### DISCUSSION TITLE
关于 deepin 系统版本发布信息公示相关事项

### DISCUSSION THREAD

**Discussion by zccrs** [linuxdeepin/developer-center#4695] [Developer Discussion | 开发者讨论]
目前初步的想法是在 wiki.deepin.org 维护类似于 https://wiki.qt.io/Qt_6.6_Release 的内容，包含下一个待发布版本的关键里程碑的时间节点，例如：

![image](https://github.com/linuxdeepin/developer-center/assets/13449038/75e420e5-0c2c-4dd9-9a1f-a88614c02cce)

一些待讨论的问题：

- [ ] 对应的wiki词条由谁维护？目前看可以是本次发布的 Release Manager
- [ ] 如果计划改变，信息更新后是否需要通知，使用哪个渠道进行通知

**Comment by Zeno-sole** [comment]
一些补充： 里程碑直接用github的milestone 就挺好，计划改变、信息更新通常是需要通知的 邮件列表通知就好了

**Comment by BLumia** [comment]
> 对应的wiki词条由谁维护？目前看可以是本次发布的 Release Manager

目前应该是对应负责发布的人员。如果后续有 RelEng 团队的话，应该由 RelEng 团队的相关负责人维护。

> 里程碑直接用github的milestone 就挺好

GitHub 的里程碑可能不太够，就上图而言，一个版本发布的过程中涉及了多个时间节点，一个里程碑应该是不够的。Project 的类甘特图的时间线视图可以起类似作用，但或许还是单页面更直观一些？

> 信息更新后是否需要通知

显然需要，因为产品、研发、测试人员可能需要酌情调整研发周期，比如更早或更晚进行版本集成，或规划一个特性是否应当涵盖在当次版本发布中。

> 使用哪个渠道进行通知

考虑到参与整个流程的主要人员是研发人员，邮件列表应该就够了。对于需要涉及到版本发布事项的非研发人员，也建议这些人员关注邮件列表即可。
